moped-editor/src/views/dev/LookupsView/settings.js

```diff
@@ -223,4 +223,22 @@ export const SETTINGS = [
       },
     ],
   },
+  {
+    key: "moped_fund_status",
+    label: "Fund Status",
+    columns: [
+      {
+        key: "funding_status_id",
+        label: "Status ID",
+      },
+      {
+        key: "funding_status_name",
+        label: "Name",
+      },
+      {
+        key: "funding_status_description",
+        label: "Description",
+      },
+    ],
+  },
```

Comment:
Might be nice to move this table into alphabetical order 📖 

<img width="1057" alt="Screenshot 2024-12-30 at 10 43 28 AM" src="https://github.com/user-attachments/assets/bc52de9b-8e8a-4435-a5a1-08f005a20347" />

Reply:
yeah! i think that is a great detail. thanks John, and could you update, @mateoclarke?